sdk/src/claim.rs

```diff
@@ -1938,12 +1938,10 @@ impl Claim {
     // Do any assertions of this type exist?
     pub fn has_assertion_type(&self, in_label: &str) -> bool {
         let (label, _) = Claim::assertion_label_from_link(in_label);
-        let found = self
-            .assertion_store
-            .iter()
-            .find(|&x| x.assertion.label().starts_with(&label));
 
-        !matches!(found, None)
+        self.assertion_store
```

Comment:
This will likely conflict with my change for Rust 1.72 compatibility because new Clippy didn't like the `!matches` construction. Your overall change looks better, so please resolve by accepting your version.

Reply:
ah, I think you're right. will change!